lablqml/lablqml.mli

```diff
@@ -64,8 +64,12 @@ module QQmlAppEngine : sig
 
   (* Use this function to get access to functions from QQmlEngine module *)
   val to_QQmlEngine : t -> QQmlEngine.t
+  val root_named: t -> string -> cppobj
 end
 
+val object_child_named: cppobj -> string -> cppobj
+val object_property_named : cppobj -> string -> cppobj
```

Comment:
You are right, the second one returns a property value. Was planning to look at types later. Maybe can do it in a following merge.